os_windows.go

```diff
@@ -94,6 +95,15 @@ func init() {
 		filepath.Join(data, "lf", "icons"),
 	}
 
+	socket, err := syscall.Socket(syscall.AF_UNIX, syscall.SOCK_STREAM, 0)
+	if err != nil {
+		gDefaultSocketProt = "tcp"
+		gDefaultSocketPath = "127.0.0.1:12345"
+	} else {
+		gDefaultSocketPath = filepath.Join(data, "lf", fmt.Sprintf("lf.%s.sock", gUser.Username))
+		syscall.Close(socket)
+	}
+
```

Comment:
Slightly pedantic, but I suggest to move this to the bottom of the `init` function after the path definitions, to maintain the same order as [`os.go`](https://github.com/gokcehan/lf/blob/master/os.go).